docs/section5/Rule5-19.md

```diff
@@ -18,34 +18,32 @@
 **Data Lookup:** Table G3.1.1-1  
 **Function Call:**  
 
-  1. get_building_segment_window_wall_areas()  
+  1. get_area_type_window_wall_areas()  
   2. data_lookup()  
   3. match_data_element()
 
 ## Rule Logic:  
 
-- Get window wall areas dictionary for B_RMR: `window_wall_areas_dictionary_b = get_building_segment_window_wall_areas(B_RMR)`
+- Get window wall areas dictionary for B_RMR: `window_wall_areas_dictionary_b = get_area_type_window_wall_areas(B_RMR)`
 
-- Get window wall areas dictionary for P_RMR: `window_wall_areas_dictionary_p = get_building_segment_window_wall_areas(P_RMR)`
+- Get window wall areas dictionary for P_RMR: `window_wall_areas_dictionary_p = get_area_type_window_wall_areas(P_RMR)`
 
 - For each building segment in the Baseline model: `for building_segment_b in B_RMR.building.building_segments:`
 
-  - Check if building segment area type is not included in Table G3.1.1-1: `if NOT data_lookup(table_G3_1_1_1, building_segment_b.area_type_vertical_fenestration):`
+  - Check if building segment area type is not included in Table G3.1.1-1: `if NOT building_segment_b.area_type_vertical_fenestration:`
 
-    - Calculate building segment window wall ratio: `building_segment_wwr_b = window_wall_areas_dictionary_b[building_segment_b][0] / window_wall_areas_dictionary_b[building_segment_b][1]`
+    - Calculate building segment window wall ratio: `area_type_wwr_b = window_wall_areas_dictionary_b["NONE"]["TOTAL_WINDOW_AREA"] / window_wall_areas_dictionary_b["NONE"]["TOTAL_WALL_AREA"]`
 
```

Comment:
This RDS should be the same as 5-18 and aggregate wall area and window area for the building segments that have the same occupancy type and only then calculate WWR. For example, if building has severall building segments with the occupancy type "multifamily", we should be verifying the totall WWR across all multifamily segments, not in each multifamily segment.